src/docfx/cli/Program.cs

```diff
@@ -89,22 +92,24 @@ private static (string command, string docset, CommandLineOptions options) Parse
             ArgumentSyntax.Parse(args, syntax =>
             {
                 // Restore command
-                // usage: docfx restore [docset] [--git-token token] [--locale locale]
                 syntax.DefineCommand("restore", ref command, "Restores dependencies before build.");
                 syntax.DefineOption("locale", ref options.Locale, "The locale of the docset to build");
                 syntax.DefineParameter("docset", ref docset, "Docset directory that contains docfx.yml/docfx.json.");
 
                 // Build command
-                // usage: docfx build [docset] [-o/--output output] [--log log] [--legacy] [--git-token token] [--locale locale]
```

Comment:
>/ usage: docfx build [docset] [-o/--output output]  [](start = 17, length = 51)

not everyone knows the syntax, why not just keep it

Reply:
Run `docfx --help` or `docfx build --help` will give you the detailed help message. 
Document it in code has extra maintenance effort and it is very likely to be out of sync. (E.g., the comment for restore has a [--git-token] in comments but that one has already been removed.
```
C:\Users\yufeih>docfx build --help
usage: docfx build [-o <arg>] [--legacy] [--github-token <arg>]
             [--locale <arg>] [--] <docset>

    -o, --output <arg>      Output directory in which to place built
                            artifacts.
    --legacy                Enable legacy output for backward
                            compatibility.
    --github-token <arg>    The GitHub token used to get contribution
                            information from GitHub API
    --locale <arg>          The locale of the docset to build
    <docset>                Docset directory that contains
                            docfx.yml/docfx.json.
```